tds/src/main/java/thredds/server/ncss/params/NcssParamsBean.java

```diff
@@ -405,6 +407,20 @@ public boolean isValidGridRequest() {
 
   public boolean intersectsTime(FeatureDataset fd, Formatter errs) throws ParseException {
     CalendarDateRange have =  fd.getCalendarDateRange();
+    try {
+      //
+      // if this is an updatable collection, check for a new CalendarDateRange (i.e. new
+      // data available).
+      // The issue is that fd does not get updated with thredds.catalog.InvDatasetFcPoint
+      //
+      // This allows requests to be fulfilled, even if the correct CalendarDateRange is
+      // not available in the catalog. Ideally these should be synced.
+      //
+      UpdateableCollection uc = (UpdateableCollection) fd;
+      have = uc.update();
+    } catch (IOException e) {
```

Comment:
Probably need to log an error on the `IOException` and catch (and swallow) any errors from casting to `UpdatableCollection`.